src/audio/volume/volume.h

```diff
@@ -112,6 +112,9 @@ struct sof_ipc_ctrl_value_chan;
 #define VOL_S16_SAMPLES_TO_BYTES(s)	((s) << 1)
 #define VOL_S32_SAMPLES_TO_BYTES(s)	((s) << 2)
 
+/** \brief PCM samples align requirement for HiFi3 an Hifi4 for volume component */
+#define VOLUME_HIFI3_HIFI4_FRAME_BYTE_ALIGN_6CH	16
```

Comment:
Comment says "PCM samples" which is a bit misleading as this is alignment in bytes (as tge define name says, FRAME_BYTE).

Reply:
Oops yes, it looks quite confusing.